### PR TITLE
Added RHOSAK quick start

### PR DESCRIPTION
**Pull request by cfchase**

### CODE REVIEW

**Comment by cfchase** [comment]
rhosak quickstart

File: install/dev-backend.sh [hunk 1]
```diff
@@ -8,4 +8,4 @@ pwd
 
 PORT=${BACKEND_DEV_PORT}
 npm install
-npm start:dev
+npm run start:dev
```
Comment (jeff-phillips-18) [inline install/dev-backend.sh:11]: Thanks for fixing these!

**Comment by cfchase** [comment]
Accompanied by sample notebooks PR https://github.com/rh-aiservices-bu/notebook-examples/pull/1.  We need to figure out where sample notebooks would live.  opendatahub-io?

@sophwats if you could take a look at the text

**Comment by cfchase** [comment]
PR for RHOSAK tile to follow

**Review by jeff-phillips-18 — APPROVED** [review]

**Pull request merged**